src/index.ts

```diff
@@ -21,6 +24,8 @@ export class DynamoDBCache implements KeyValueCache {
   private client: DynamoDB.DocumentClient;
   private tableName: string;
   private partitionKeyName: string;
+  private secondaryKeyName: string | undefined;
```

Comment:
Why change the behavior here for the secondary key? Why not give it a default like the primary?

Reply:
Some tables won't have a secondary key configured at all. All tables must have at least a partition key, but a sort key is not required. So a "default" makes sense for the partition key, but for the sort key we want to allow it to be undefined.

- table with only partition key - getItem with `partitionKeyName`
-  table with partition & sort key - getItem with `partitionKeyName` and `secondaryKeyName`

Reply:
I'm going to rename `secondary` to `sort`. Not sure what I was thinking there.